OSDOCS-16950#CQA work Stor6 - LVMS snapshots and clones (#116575)
openshift/openshift-docs: modules/lvms-creating-volume-snapshots.adoc

[id="lvms-creating-volume-snapshots_{context}"]
= Creating volume snapshots

[role="_abstract"]
Create volume snapshots to capture point-in-time copies of persistent volume claims (PVCs) for data backup or recovery purposes by creating a `VolumeSnapshot` object, based on the available thin pool capacity and over-provisioning limits.

> You can create volume snapshots based on the available capacity of the thin pool and the over-provisioning limits.

Reviewer: Redundant. Consider removing.

apiVersion: snapshot.storage.k8s.io/v1
kind: VolumeSnapshot
metadata:
  name: lvm-block-1-snap
spec:
  source:
    persistentVolumeClaimName: lvm-block-1
  volumeSnapshotClassName: lvms-vg1
----
+
* `metadata.name`: Specifies a name for the volume snapshot.

> * `spec.persistentVolumeClaimName`: Specifies the name of the source PVC. {lvms} creates a snapshot of this PVC.

Reviewer: ```suggestion
* `spec.source.persistentVolumeClaimName`: Specifies the name of the source PVC. {lvms} creates a snapshot of this PVC.
```

> * `spec.volumeSnapshotClassName`: Set this field to the name of a volume snapshot class.

Reviewer: ```suggestion
* `spec.volumeSnapshotClassName`: Specifies the name of a volume snapshot class.
```
For consistency. Consider making the other lists consistent as well.
